doc/whats_new.rst

```diff
@@ -68,7 +68,8 @@ Enhancements
 
    - Added ``classes_`` attribute to :class:`model_selection.GridSearchCV`
      that matches the ``classes_`` attribute of ``best_estimator_``. :issue:`7661`
-     by :user:`Alyssa Batula <abatula>` and :user:`Dylan Werner-Meier <unautre>`.
+     and :issue:`8295` by :user:`Alyssa Batula <abatula>`,
```

Comment:
Maybe add the class `grid_search.GridSearchCV`, `grid_search.RandomizedSearchCV`  and `model_selection.RandomizedSearchCV` to make this entry more exact.